workflows/files/ZWEWRF06.xml

```diff
@@ -0,0 +1,247 @@
+                    <description>Checking VOLUME and Storage Class</description>
+                    <expression>(${instance-xmem_volume} &amp;&amp; ${instance-xmem_volume} != "") || (${instance-xmem_storclas} &amp;&amp; ${instance-xmem_storclas} != "")</expression>
+                    <targetState>skipped</targetState>
+                </extendStateExpression>
+            </targetStateSet>
+        </condition>
+        <!--variableValues-->
+        <instructions substitution="false">Run this step to retrieve the actual VOLUME of the LOADLIB</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <template>
+            <inlineTemplate substitution="true">set -e
+mkdir -p ${instance-temp_folder}
+listing=$(tsocmd "LISTDS ('${instance-zowe_dsn_prefix}.LOADLIB')")
+volume=$(echo $listing|tr ' ' '\n'|sed -n "/--VOLUMES--/{n;p;}")
+echo "xmem_volume=$volume"&gt;${_output}</inlineTemplate>
+            <submitAs maxRc="0">shell-JCL</submitAs>
+            <maxLrecl>1024</maxLrecl>
+            <output needResolveConflicts="false" substitution="true">${instance-temp_folder}/volume</output>
+        </template>
+    </step>
+    <step name="apf_auth" optional="false">
+        <title>APF Authorize Load Library</title>
+        <description>This step will dynamically APF authorize the Cross Memory Server LOADLIB</description>
+        <prereqStep name="copy_artifacts"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="true">Run this step to dynamically APF authorize the load library.
+If you want the APF, to be permanent, add following library to your system authorization: ${instance-zowe_dsn_prefix}.LOADLIB</instructions>
+        <weight>1</weight>
+        <skills>z/OS System Programmer</skills>
+        <autoEnable>true</autoEnable>
+        <canMarkAsFailed>false</canMarkAsFailed>
+        <rest>
+            <httpMethod>PUT</httpMethod>
+            <uriPath substitution="true">/zosmf/restconsoles/consoles/${_step-stepOwnerUpper}</uriPath>
+            <requestBody substitution="true">#if(${instance-xmem_storclas} and $!{instance-xmem_storclas} != "")
+#set($dasd = "SMS")
+#else
+#set($dasd = "VOLUME=${instance-xmem_volume}")
+#end
+{"cmd":"SETPROG APF,ADD,DSNAME=${instance-zowe_dsn_prefix}.LOADLIB,${dasd}"}</requestBody>
+            <expectedStatusCode>200</expectedStatusCode>
+            <!-- REST mappings -->
+        </rest>
+    </step>
+    <step name="start_xmem" optional="false">
+        <title>Start the Cross Memory Server server</title>
+        <description>This step starts the Cross Memory Server</description>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="false">Run this step to start the Cross Memory Server.
+
+You can also do it manually with the following command:
+/S ZWESISTC,REUSASID=YES
```

Comment:
the leading / is only valid inside SDSF, so I would say 
... the following operator command (prefix it with / when issued from the SDSF command line):
S ZWESISTC,REUSEASID=YES